### ISSUE TITLE
Problem testing the ent-elk crud example

### ISSUE THREAD

**Issue by Exnoic** [masseelch/elk#47]
Hi,

I was following https://entgo.io/blog/2021/07/29/generate-a-fully-working-go-crud-http-api-with-ent

When i try to run the server i get this error

go run -mod=mod main.go

```
# command-line-arguments
.\main.go:32:26: "elk-example/ent/http".NewPetHandler(c, l).Mount undefined (type *"elk-example/ent/http".PetHandler has no field or method Mount)
.\main.go:32:36: undefined: "elk-example/ent/http".PetRoutes
```

Here is the github repo https://github.com/Exnoic/ent-elk-example

Please tell me if you need any other information!

Best regards,
Henrik

**Comment by masseelch** [comment]
Hey @Exnoic, thanks for reaching out.

 I just had a look: The current master of `elk-example` is working as it should. 

The blog post you are reading is outdated (sadly). 

Your repo uses `elk` in v0.5.2 while the example uses v0.2.1. So there have been some changes made.

To fix your Code: 
```shell
go get -u entgo.io/ent@master # to fix a bug in 0.9.2 you are using
```
And  make the following change to your `main.go`:

``diff
- elk.NewPetHandler(c, l).Mount(r, elk.PetRoutes)
+ elk.NewPetHandler(c, l).MountRoutes(r)
```

**Comment by Exnoic** [comment]
Thank you for the help it now works.

I will close this now.
